logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/SessionManagerExt.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.feature
+
+import com.wire.kalium.logic.data.auth.login.ProxyCredentials
+import com.wire.kalium.logic.di.MapperProvider
+import com.wire.kalium.network.session.SessionManager
+
+internal fun SessionManager.getProxyCredentials(): ProxyCredentials? =
+    MapperProvider.sessionMapper().fromDTOToProxyCredentialsModel(proxyCredentials())
```

Comment:
since its an extension function the Proxy Credentials is suggest to have the naming only like `fromDTO`; what do you think?